packages/web/README.md

```diff
@@ -3,3 +3,72 @@ Natura Design System - Web Components
 
 ![Travis (.org)](https://img.shields.io/travis/natura-cosmeticos/natds.svg)
 [![Known Vulnerabilities](https://snyk.io/test/github/natura-cosmeticos/natds/badge.svg?targetFile=package.json)](https://snyk.io/test/github/natura-cosmeticos/natds?targetFile=package.json)
+
+---
+
+## Install
+
+```
+$ yarn add @naturacosmeticos/natds-web
+```
+
+## Importing
+
+```
+import { Chip } from '@naturacosmeticos/natds-web'
```

Comment:
Seria mais para esse início não é? Pq acho que não dará pra colocar o tanto de exemplos de imports...tvz melhor tirar esse bloco, não sei :thinking:

Reply:
é um exemplo, todos os componentes vão ser importados de maneira igual, só um funciona